proxy/proxy.go

```diff
@@ -57,9 +57,9 @@ func NewProxy(c Config) (*Proxy, error) {
 	p.client = client
 
 	if !p.WithoutDNS {
-		dockerBridgeIP, err := callWeave("docker-bridge-ip")
+		dockerBridgeIP, stderr, err := callWeave("docker-bridge-ip")
 		if err != nil {
-			return nil, err
+			return nil, fmt.Errorf(string(stderr))
```

Comment:
So previously this was returning `err`. What's the rationale for choosing one over the other? Should we perhaps return a combination of both?

Same applies at the other `callWeave` site.